scripts/utils/gnosis_safe_server_interactions.js

```diff
@@ -142,9 +143,52 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     return 1 + nonce
   }
 
+  /**
+   * Either checks if the transaction exists on the web interface
+   * or create a new transaction, depending on the first flag.
+   *
+   * @param {boolean} verifyOnly Whether to verify if the transaction exists
+   * or creating a new one.
+   * @param {SmartContract} masterSafe The Safe used to execute the
+   * transaction.
+   * @param {number} nonce The nonce used by the transaction.
+   * @param {Transaction} transaction The transaction to be sent.
+   * @param {string} network Either rinkeby, xdai or mainnet.
+   * @param {boolean} executeOnchain Whether the transaction should be
+   * executed immediately onchain or just created in the web interface.
+   * @param {boolean} mustPromptUser Whether to prompt the user with a
```

Comment:
I would be a bit happier if this were simply called `promptUser`, but I guess this would conflict with the function name.

Reply:
I agree with you, so much so that this was the original name I gave it. Then I realized that this was the name of the function prompting the user...